.gitignore

```diff
@@ -2,5 +2,4 @@ node_modules/
 dist/
 .idea
 .DS_STORE
-bundle.js
```

Comment:
There is no `bundle.js` generated in this project.  The bundle generated is `dist/index_bundle.js` and is already ignored by the `dist/` entry on line 2.